src/customJs/payuButtonTool/index.js

```diff
@@ -7,7 +7,7 @@ import { intl } from '../localization';
 export const getPayuButtonToolConfig = () => ({
   name: 'payu_button_tool',
   label: intl.formatMessage({ id: 'payu_button' }),
-  icon: 'fa-share-alt',
+  icon: process.env.PUBLIC_URL + '/assets/Payu_Button.svg',
```

Comment:
no podemos importar el svg con webpack de manera que se incluya en el CDN y el nombre se genere con un hash?

Reply:
Remenber, this is not a CRA environment, in this project we have 2 webpacks config, the first is for Unlayer and the second is just for local environment (CRA), to get the the assets hash in the build we need do all in webpack. I think that is a lot efforce.